hopsworks-web/yo/app/views/storageConnectorViewInfo.html

```diff
@@ -143,6 +143,25 @@ <h2 class="modal-title">Storage Connector Details: {{storageConnectorViewInfoCtr
                     </td>
                 </tr>
 
+
+                <tr class="fs-table" ng-if="!storageConnectorViewInfoCtrl.settings.s3IAMRole">
```

Comment:
remove this check. encryption information should be shown regardless of the iam role